go.mod

```diff
@@ -37,6 +37,7 @@ require (
 	github.com/google/gofuzz v1.2.0
 	github.com/google/uuid v1.6.0
 	github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0
+	github.com/k-orc/openstack-resource-controller v0.0.0-00010101000000-000000000000
```

Comment:
Is there not a tag version we could point to here?

Reply:
not yet. Also as you can see line 273 (formerly 272) of this same `go.mod`, the requirement is `replace`d because the code for openstack-resource-controller temporarily lives in the cluster-api-provider-openstack repository

Reply:
Ok, just wanted to double check